2018-09-15-scipy-mm-nc/proposal/short.md

```diff
@@ -0,0 +1,13 @@
+The Short Description:
+
+This talk covers a tool developed for fast, interactive, browser-based
```

Comment:
"This talk covers a Jupyter widget built using Rust and WebAssembly for interactive, browser-based visualization of datasets with yt, a python package..."